packages/cli/src/commands/experimental/templates/docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.8"
+
+services:
+  api:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      target: api_serve
+    ports:
+      - "8911:8911"
+    depends_on:
+      - db
+    environment:
+      - DATABASE_URL=postgresql://redwood:redwood@db:5432/redwood
+      - TEST_DATABASE_URL=postgresql://redwood:redwood@db:5432/redwood_test
+      - SESSION_SECRET=super_secret_session_key_change_me_in_production_please
+
+  web:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      target: web_serve
+    ports:
+      - "8910:8910"
+    depends_on:
+      - api
+    environment:
+      - API_HOST=http://api:8911
+
+  db:
+    image: postgres:15-bookworm
+    environment:
+      POSTGRES_USER: redwood
+      POSTGRES_PASSWORD: redwood
+      POSTGRES_DB: redwood
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
+
+  # Use the console to run commands in the container:
+  #
+  # ```
+  # docker compose run --rm -it console /bin/bash
+  # ```
+  console:
+    user: root
```

Comment:
Todo: add back some comments here that I removed because I didn't understand them